hawk/src/main/java/com/orhanobut/hawk/SqliteStorage.java

```diff
@@ -92,10 +93,15 @@ public SqliteHelper(Context context, String dbName) {
 
     public synchronized boolean put(String key, String value) {
       SQLiteDatabase db = this.getWritableDatabase();
-      db.execSQL("INSERT OR REPLACE INTO " + TABLE_NAME +
-          " (" + COL_KEY + ", " + COL_VALUE + ") " +
-          " VALUES('" + key + "', '" + value + "')");
-      db.close();
+      try {
+        db.execSQL("INSERT OR REPLACE INTO " + TABLE_NAME +
+                " (" + COL_KEY + ", " + COL_VALUE + ") " +
+                " VALUES('" + key + "', '" + value + "')");
+      } catch (SQLException ignored) {
+        return false;
```

Comment:
I saw the signature of method db.execSQL throws SQLException
```
public void execSQL(String sql) throws SQLException {
        executeSql(sql, null);
    }
```
But wider exception catch is better ,OK, I will modify that and launch another PR